Added test class. closes #585

ravikamaraj opened the pull request:
@wslulciuc , can you review. thanks

wslulciuc commented:
Great to see more tests for our API models! Recently, we've simplified the way we write tests for our models. Mind updating the PR to follow these tests?

- [`DatasetRequestTest`](https://github.com/MarquezProject/marquez/blob/master/src/test/java/marquez/api/models/DatasetRequestTest.java)
- [`DatasetResponseTest`](https://github.com/MarquezProject/marquez/blob/master/src/test/java/marquez/api/models/DatasetResponseTest.java)
- [`DatasetsResponseTest`](https://github.com/MarquezProject/marquez/blob/master/src/test/java/marquez/api/models/DatasetsResponseTest.java)

wslulciuc commented on src/test/java/marquez/api/models/ApiModelGenerator.java:
```diff
@@ -126,4 +128,16 @@ public static JobResponse newJobResponse(final Boolean hasDescription) {
+        newDatasourceName().getValue(),
+        newIsoTimestamp(),
+        newDatasourceUrn().getValue(),
+        newConnectionUrl().getRawValue());
```
💯

ravikamaraj commented:
> Great to see more tests for our API models! Recently, we've simplified the way we write tests for our models. Mind updating the PR to follow these tests?

done, can you review. Also added copyright information in DatasetRequest class
